examples/v2/project_creation/project.py

```diff
@@ -98,3 +101,35 @@ def GenerateConfig(context):
     })
 
   return {'resources': resources}
+
+def MergeCallingServiceAccountWithOwnerPermissinsIntoBindings(env, properties):
+  """ A helper function that merges the acting service account of the project
+      creator as an owner of the project being created
+  """
+  service_account = ('serviceAccount:{0}@cloudservices.gserviceaccount.com'
+                     .format(env['project_number']))
+  set_creator_sa_as_owner = {
+      'role': 'roles/owner',
+      'members': [
+          service_account,
+      ]
+  }
+  if ('iam-policy' not in properties or
```

Comment:
I somewhat agree with krwenholz. This should probably not have the test for 'bindings'. As that makes the assumption that there is nothing in the IAM policy except bindings, which may not always be true. 

There will need to be additional logic to insert a 'bindings' if it doesn't exist. Then the existing merge logic below will work.